src/framework/core/js/FluidView.js

```diff
@@ -233,120 +316,278 @@ var fluid_4_0_0 = fluid_4_0_0 || {}; // eslint-disable-line no-redeclare
+    fluid.checkMaterialisedElement = function (element, selectorName, that) {
+        if (!element || !element.length) {
+            fluid.fail("Could not locate element for selector " + selectorName + " for component " + fluid.dumpComponentAndPath(that));
         }
-        that.dom = fluid.createDomBinder(that.container, selectors || that.options.selectors || {});
-        that.locate = that.dom.locate;
-        return that.dom;
     };
 
-    // DOM Utilities.
-
-    /**
-     * Finds the nearest ancestor of the element that matches a predicate
-     * @param {Element} element - DOM element
-     * @param {Function} test - A function (predicate) accepting a DOM element, returning a truthy value representing a match
-     * @return {Element|undefined} - The first element parent for which the predicate returns truthy - or undefined if no parent matches
-     */
-    fluid.findAncestor = function (element, test) {
-        element = fluid.unwrap(element);
-        while (element) {
-            if (test(element)) {
-                return element;
+    fluid.domMaterialiserManager = fluid.makeDomMaterialiserManager();
+
+    // Passive - pushes out to single-arg jQuery method, active in acquiring initial markup value for booleanAttr
+    fluid.materialisers.domOutput = function (that, segs, type, options) {
+        fluid.freezeRecursive(segs);
+        var selectorName = segs[1];
+        var listener = function (value) {
+            if (that.dom) {
+                var element = that.dom.locate(selectorName);
+                fluid.checkMaterialisedElement(element, selectorName, that);
+
+                if (type === "jQuery") {
+                    var model = {
+                        value: value,
+                        segs: segs
+                    };
+                    var args = options.makeArgs ? options.makeArgs(model) : [model.value];
+                    element[options.method].apply(element, args);
+                } else if (type === "booleanAttr") {
+                    if (value === undefined) {
+                        var markupValue = !!element.attr(options.attr);
+                        that.applier.change(segs, options.negate ? !markupValue : markupValue);
+                    } else {
+                        var attrValue = options.negate ? !value : value;
+                        if (attrValue) {
+                            element.attr(options.attr, options.attr);
+                        } else {
+                            element.removeAttr(options.attr);
+                        }
+                    }
+                }
             }
-            element = element.parentNode;
-        }
+        };
+        //        fluid.pushArray(fluid.domMaterialiserManager.idToModelListeners, that.id, listener);
```

Comment:
should this be removed?

Reply:
This is in-progress functionality for what is really a "sneak peek" implementation of https://issues.fluidproject.org/browse/FLUID-6683 which got put into the Infusion 4.x work since it didn't cause any disruption to existing code, but is still quite unstable. It's probably best to leave the comment as an aid to memory about how this impl might get improved.